packages/benchmarks/src/nlPromptResponse/loadDocs100Dataset.ts

```diff
@@ -0,0 +1,79 @@
+import { LanguageModel } from "ai";
+import {
+  tagifyMetadata,
+  classifyMongoDbMetadata,
+} from "mongodb-rag-core/mongoDbMetadata";
+import { NlPromptResponseEvalCase } from "./NlQuestionAnswerEval";
+import { parse } from "csv-parse/sync";
+import fs from "fs";
+import path from "path";
+
+type Docs100Row = {
+  Question: string;
+  TeamtoReview: string;
+  Author: string;
+  ExpectedSources: string; // This will be a comma-separated string of URLs
+  IdealAnswer: string;
+  DateCreated: string; // Assuming date is stored as a string
+  AdditionalNotes: string;
+};
+
+export async function parseDocs100QARow(
+  row: Docs100Row,
+  model: LanguageModel
+): Promise<NlPromptResponseEvalCase> {
+  const tags = ["docs", "docs_100"];
```

Comment:
Per our brief discussion on Slack, can we add a tag like `"verified_response"` to all of the cases in these datasets. i.e. indicate that "this expected response was verified and approved by a SME".